public/js/core.js

```diff
@@ -1,21 +1,17 @@
 var tarchive = angular.module('tarchive', ["APIService", "ngTable", "ngSanitize"]);
 
-tarchive.controller('CoreController', ['$scope', '$filter', 'API', 'NgTableParams', function($scope, $filter, API, NgTableParams) {
+tarchive.config(['$compileProvider', function( $compileProvider ) {
+    $compileProvider.aHrefSanitizationWhitelist(/^\s*(https?|http|ftp|mailto|data|chrome-extension):/);
```

Comment:
And ofcourse the the `http*` ones! I tried removing them, resulting in the `t.me` links being flagged unsafe, so I only left in `https`, `http`, and `data`. All we need right now.

But thanks for the feedback!